src/pages/hackathon/breakout.tsx

```
@@ -0,0 +1,382 @@
+//               <AccordionContent className="px-4 pt-3 text-sm text-slate-700 sm:text-base [&_a]:text-blue-700">
+//                 <div dangerouslySetInnerHTML={{ __html: f.answer }} />
+//               </AccordionContent>
+//             </AccordionItem>
+//           ))}
+//         </Accordion>
+//       </div>
+//     </div>
+//   );
+// }
+
+export const getServerSideProps: GetServerSideProps = async ({}) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)